### ISSUE TITLE
Unable to get Colors to show

### ISSUE THREAD

**Issue by meyerder** [briangann/grafana-datatable-panel#3]
I am unable to get **any** of the colors to show in the datatable panel

Grafana 4.3.0-pre1

successful
npm install
bower install

![capture](https://cloud.githubusercontent.com/assets/28714840/26117096/f2d231a2-3a2a-11e7-9419-c143369ba2fe.PNG)

![capture](https://cloud.githubusercontent.com/assets/28714840/26116984/a87a5558-3a2a-11e7-91c9-b9a8adaf9830.PNG)

**Comment by briangann** [comment]
These are column styles.  If you want to set thresholds, I'd set the Column Name in the "Name or Regex" field, then set the threshold values.  Typically you have to select invert also. It will set the color for the value in your example.  You can tell it to set the cell to that color, or the row.

You might have sort this out already though :)

**Comment by meyerder** [comment]
Yea I was hoping to get something that does ROW based but not a column...

**Comment by briangann** [comment]
So you want to set the color based on the name of the row versus the column?

I can add that as an option :)

**Comment by meyerder** [comment]
Yes PLEASE....  in my case I have the following.  I hope this makes sense

-----------Data
<Average Time to repair> <Company 1> <Company 2> <Company 3>
<Amount of issues> <Company 1> <Company 2> <Company 3>
<Average Impact>  <Company 1> <Company 2> <Company 3>


Where the following exists
Average time to repair if > 20 make it red
Amount of issues if > 10 make it Orange if > 20 then Red
Average Impact if > 100 then red

**Comment by meyerder** [comment]
If you wanna get really fancy... impliment the html hotlinks inside of
https://github.com/dmitrios/grafana-plugins-reftable

[image: Inline image 1]

**Comment by briangann** [comment]
I'll take a look at reftable to see what it does.  

I did put in html linking into the table options so any cell that has a link will click through.

![image](https://user-images.githubusercontent.com/7364245/29083477-7e7a0776-7c2e-11e7-95e9-8d48785de11b.png)

**Comment by meyerder** [comment]
Looks like I will have to upgrade as I dont have that option... says it is
up to date but Ill download from github

Version 0.0.1

**Comment by briangann** [comment]
yeah v0.0.3 is in progress, and v0.0.2 wasn't published :(

**Comment by meyerder** [comment]
BTW if I am understanding your url item the other package allows to build a
dynamic link based on the cell value (works well for a monitoring dashboard
with links directly to the site details)

**Comment by timrsa** [comment]
@briangann Thank you for making a table that you can search values in, it is helpful. One thing I would like to get working would be to be able to assign a colour to a value in a column. Specifically What I am doing is getting a list of VM's running in a various DC's, I collate that info into a MySQL table and from Grafana I am doing a Select. The one value I would like to change the colour is the VM powerstate, this currently stands as PoweredOn / PoweredOff. I would like to use the same functionality as thresholds have with numerical values, if I do a value mapping I can sort of get it to work but then I have to publish those values as 1 and 0. Got any clever workarounds for me? :) Thanks for the work on the plugin again, very cool.
